### ISSUE TITLE
Inline messages reading order is unconventional

### ISSUE THREAD

**Issue by Magsu** [sl-design-system/components#1558]
### Provide a general summary of the issue here

Some inline messages have a reading order that does not represent the structure of information. This can frustrate users or make the user think they’ve missed content, that is not screen reader accessible. Furthermore, nonlinear reading orders break the information structure, thus making information hard to understand.

### 🤔 Expected Behavior?

When entering a section on a page the user would be first directed to the main element (in this case heading or body text depending if a heading is available). After the user has gathered the information in the message, the user is prompted to act on or close the message. Linear reading order would be heading -> body text -> action button -> close button

### 😯 Current Behavior

Currently the reading order varies between different types.
In the image example the user using a screen reader is prompted to act on or close the message before having information what they’re acting on or closing. The reading order is currently action button -> close button -> body text.
![image](https://github.com/user-attachments/assets/642a5d70-65a9-4f55-be1c-2ea45117ec8e)

In the below example a screen reader user would get some information for performing action or closing the message before. The reading order for a screen reader is heading -> action button -> close button -> body text.
![image](https://github.com/user-attachments/assets/f03e36fe-9003-4326-b361-79f5f0784a81)

### 💁 Possible Solution

Currently the elements in the dom are as follows:
![image](https://github.com/user-attachments/assets/3a474aad-d29a-4d26-9d22-7bd46b93ae85)

Moving the div with part=”content” upper in DOM takes care of this issue. This should be after the heading (part="title") and before the buttons (part="action").
![image](https://github.com/user-attachments/assets/9d00d5be-18be-4b0c-a455-ebdca60d0bb0)

### 🖥️ Steps to Reproduce

1. Open a page with a Inline message (e.g. [Inline message Overflow](Open https://storybook.sanomalearning.design/?path=/story/feedback-status-inline-message--overflow)
2. Start a screen reader 
3. Use the commands to move forward and backward in the element

### What browsers are you seeing the problem on? (only for bugs in code)

Chrome

### What operating system are you using?

Windows 10

### 👤 Your name

Valtteri Järvinen

### 🧢 Your product/team

External

### 🎨 Your Theme(s)

Sanoma Learning
